app/src/main/res/values/dimens.xml

```diff
@@ -36,4 +36,10 @@
   <dimen name="tag_width">50dp</dimen>
   <dimen name="tag_margin">6dp</dimen>
   <dimen name="tag_text_size">10sp</dimen>
+
```

Comment:
Please, check the latest commits. I tried to change the Icon to text, but it did not look as good.